app/services/idv/steps/document_capture_step.rb

```diff
@@ -1,7 +1,22 @@
 module Idv
   module Steps
     class DocumentCaptureStep < DocAuthBaseStep
+      IMAGE_UPLOAD_PARAM_NAMES = %i[
+        front_image back_image selfie_image front_image_data_url back_image_data_url
+        selfie_image_data_url
+      ].freeze
+
       def call
+        if request_includes_images?
```

Comment:
`#call` is doing a lot of work now. If the request includes params, it tries to process those params. If the request does not have params, it tries load up a stored response. It'd be nice to have different actions for each of these. Unfortunately the flow state machine constrains us a little bit.